internal/js/modules/k6/browser/browser/frame_mapping.go

```diff
@@ -45,18 +45,23 @@ func mapFrame(vu moduleVU, f *common.Frame) mapping {
 				return f.Content() //nolint:wrapcheck
 			})
 		},
-		"dblclick": func(selector string, opts sobek.Value) *sobek.Promise {
+		"dblclick": func(selector string, opts sobek.Value) (*sobek.Promise, error) {
+			popts := common.NewFrameDblClickOptions(f.Timeout())
```

Comment:
Just as a comment, in this scope, what is the meaning of `pops`? Parsed options? It sounds a bit ambiguous to me. We should change to something else where it is or Frame Options or the specific DB Options, which converts to `fopts` or `dbopts`.

Of course, all of this, not in this pull request.

Reply:
It is parsed options and was what was used in the code that we pull out so it isn't in a separate goroutine.